docs/README.md

```diff
@@ -88,10 +156,4 @@ Refer to the following Zowe Client SDK reference guides for information about th
 
 ## Feedback
 
-Your feedback is important to us. Tell us what you think about the experience by creating an issue in GitHub or contacting the team on Slack.
-
-[Create Issue in GitHub](https://github.com/zowe/docs-site/issues) | [Ask Question on Slack](https://slack.openmainframeproject.org/)
-
-We review your feedback and make improvements to provide better content experience. Take a look at what we have been doing to address feedback.
-
-[Progress Update](https://github.com/zowe/docs-site/wiki/User-feedback-and-content-update)
+Your feedback is important to us. Tell us what you think about the experience by [creating an issue in GitHub](https://github.com/zowe/docs-site/issues) or [connecting with the community on Slack](https://slack.openmainframeproject.org/).
```

Comment:
We should maybe also mention the possibility to create an issue in community repo.